### DISCUSSION TITLE
Notebooks will load with atom but not VSCode

### DISCUSSION THREAD

**Discussion by Tim-Gaede** [fonsp/Pluto.jl#1049] [Questions about using Pluto]
UPDATE:

The problem originates from a **try** in WebServer.jl
**responsefunc(ClientRequest(session, notebook, body, Initiator(client, request_id)))**

That fails in VSCode.

I can still launch Pluto from atom and my Linux terminal (I'm a Linux noob and not too technical).

So I'm not too worried about temporarily lacking a VSCode option.

**Comment by fonsp** [comment]
Can you describe the problem in a way that we can reproduce it?

**Comment by Tim-Gaede** [comment]
I have no trouble getting to the **welcome to Pluto.jl** page: 

In the REPL:

<hr>

Opening http://localhost:1235/?secret=lCXfJ65r in your default 

Press Ctrl+C in this terminal to stop Pluto

<hr>

The problem starts when I attempt to create a new session or resume a session.
For example, when I click the new notebook link:

In the REPL:

<hr>

Opening in existing browser session.
ERROR: julia: -t,--threads=<n> must be an integer >= 1
┌ Warning: Response function to message of type complete failed
└ @ Pluto ~/.julia/packages/Pluto/bn5CN/src/webserver/WebServer.jl:370
┌ Warning: Reading WebSocket client stream failed for unknown reason:
│   parentbody =
│    Dict{Any, Any} with 5 entries:
│      "client_id"   => "1rnvxdz"
│      "body"        => Dict{Any, Any}("query"=>"sq")
│      "request_id"  => "hc5568"
│      "notebook_id" => "e96bc8ea-9506-11eb-0a46-8bea9e615…
│      "type"        => "complete"
│   exception =
│    TaskFailedException
│    Stacktrace:
│      [1] wait
<hr>

etc.

Thanks for your reply.

**Reply by fonsp** [reply]
Can you give us a step-by-step instruction on how to reproduce your problem? What is the problem _exactly_? What did you expect, how is it different? Maybe you can record a video of your screen where you clearly show the steps that you take, and then show *all* terminal output if an error occurs?
